contracts/stellar-interchain-token-service/src/tests/flow_limit.rs

```diff
@@ -353,77 +353,3 @@ fn add_flow_out_fails_on_exceeding_flow_limit() {
         ContractError::FlowLimitExceeded
     );
 }
-
-#[test]
-fn add_flow_fails_on_flow_comparison_overflow() {
```

Comment:
old test is being removed, but you are not testing the new behaviour. This needs some tests to ensure overflow is being handled correctly